### PR TITLE
JS-103 (JS-347) JavaScript Functions: Arrow Functions

### PR DESCRIPTION
**Pull request by nicoRDL**
## Related Ticket

https://codeandtheory.atlassian.net/browse/JS-347

## Description 

- completed Arrow Function tutorial
- completed all tutorial quizzes

## Steps to Validate
1. Open the index file, make sure the random.js is added
2. Open the dev tools console; a random number should appear based on the 2 function calls

## Risks
N/A

### CODE REVIEW

File: javaScriptFunctions_arrowFunctions/codeChallenge/random.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  if(high > low) {
+    const randomNum = Math.floor(Math.random() * (high - low + 1)) + low;
+    return randomNum;
+  } else {
+    return 'Invalid Input!';
+  }
+};
+
+console.log(generateRandomNumber(10, 7));
+console.log(generateRandomNumber(300, 1));
```
Comment (lycoangelo) [inline javaScriptFunctions_arrowFunctions/codeChallenge/random.js]: No newline at EOF.

File: javaScriptFunctions_arrowFunctions/codeChallenge/random.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+function generateRandomNumber (high, low) {
+  if(high > low) {
```
Comment (lycoangelo) [inline javaScriptFunctions_arrowFunctions/codeChallenge/random.js]: Nonstandard spacings:
- There should be no space after the function name and the parameters parenthesis.
- There should be a space right after the `if`

File: javaScriptFunctions_arrowFunctions/index.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <script src="codeChallenge/random.js"></script>
+</body>
+</html>
```
Comment (lycoangelo) [inline javaScriptFunctions_arrowFunctions/index.html]: No newline at EOF.

**Review by lycoangelo — CHANGES_REQUESTED** [review]

**Comment by nicoRDL** [comment]
@lycoangelo change requests and improvements done. PR updated for review

6529ca27daf93d8497c406e7617fe5fe5ea3a802

**Review by lycoangelo — APPROVED** [review]

File: javaScriptFunctions_arrowFunctions/codeChallenge/random.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+function generateRandomNumber( high, low ) {
```
Comment (lycoangelo) [inline javaScriptFunctions_arrowFunctions/codeChallenge/random.js]: @nicoRDL, you can also remove the spaces after the opening parenthesis and before the closing parenthesis:
`function generateRandomNumber(high, low)`

**Review by lycoangelo — CHANGES_REQUESTED** [review]

File: javaScriptFunctions_arrowFunctions/codeChallenge/random.js (hunk 4 above)
Comment (nicoRDL) [reply javaScriptFunctions_arrowFunctions/codeChallenge/random.js]: PR updated for review @lycoangelo 

https://github.com/nicoRDL/dotTheory-JS/pull/10/commits/ee1d5a4e3fecc6458bc21b11ce52ee189b8cc7de

**Review by lycoangelo — APPROVED** [review]

**Pull request merged**